modules/get-started/pages/rpk-quickstart.adoc

```diff
@@ -0,0 +1,244 @@
+= Redpanda CLI Quickstart
+:page-categories: rpk
+:description: pass:q[Quickly become familiar with rpk commands for basic Redpanda tasks, including creating, producing to, describing, and deleting topics, as well as consuming records and managing consumer groups.]
+// Do not put page aliases in the single-sourced content
+// tag::single-source[]
+
+This guide shows how to run the Redpanda CLI, `rpk`, for basic Redpanda tasks, including creating, producing to, describing, and deleting topics, as well as consuming records and managing consumer groups. Follow these examples to quickly become familiar with `rpk` commands.
+
```

Comment:
The intro should say something about make sure you have rpk installed, or a prereq to install rpk with link: https://deploy-preview-935--redpanda-docs-preview.netlify.app/current/get-started/rpk-install/